src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -0,0 +1,177 @@
+                                    <label className="label-component details">Street 2</label><br /><br />
+                                    <label className="label-component details">City</label><br /><br />
+                                    <label className="label-component details">State</label><br /><br />
+                                    <label className="label-component details">Zip</label><br /><br />
+                                    <div className="scs-spacing" />
+                                    <label className="label-component details">Phone</label><br /><br />
+                                    {this.props.account.numVolunteers ? 
+                                        <label className="label-component details">Volunteers</label>
+                                        :
+                                        <span />
+                                    }
+                                    {this.props.account.deliveryNotes ? 
+                                        <div>
+                                            <label className="label-component details">Notes</label><br /><br />
+                                        </div>
+                                        :
+                                        <span />
+                                    }
+                                    {this.props.account.accountType === AccountType.RECEIVING_AGENCY ? 
+                                        <label className="label-component details">Emergency Pickup</label>
+                                        :
+                                        <span />
+                                    }
+                                </div>
+                                
+                                <div className="editing-child-2">
+                                    <input name="name" type="text" className="form-input" defaultValue={this.props.account.name} /><br /><br />
+                                    <div className="scs-spacing" />
+                                    <input name="street1" type="text" className="form-input" defaultValue={this.props.account.address.street1} /><br /><br />
+                                    <input name="street2" type="text" className="form-input" defaultValue={this.props.account.address.street2} /><br /><br />
+                                    <input name="city" type="text" className="form-input" defaultValue={this.props.account.address.city} /><br /><br />
+                                    <input name="state" type="text" className="form-input" defaultValue={this.props.account.address.state} /><br /><br />
+                                    <input name="zip" type="text" className="form-input" defaultValue={this.props.account.address.zipcode} /><br /><br />
+                                    <input name="phone" type="tel" pattern={StringFormat.PHONE} className="form-input" defaultValue={this.props.account.organizationPhone} /><br /><br />
```

Comment:
remove

Reply:
Fixed.

Reply:
I have added the an input field for office number instead